### ISSUE TITLE
(When Desktop 83 is no longer supported) Remove opt out targeting

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#5505]
When Desktop 83 is no longer a supported version for new experiments, we should remove the opt-out targeting expression added in https://github.com/mozilla/experimenter/commit/949df8924f51a5bc713a63ec52640ec8c92b1f88 since it will no longer be necessary.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-588)

**Comment by jaredlockhart** [comment]
We can do this now!

**Comment by iamrahul823** [comment]
Hello @yashikakhurana. I am an Outreachy applicant and I would love to participate in this experimenter project by working on this issue.

**Comment by yashikakhurana** [comment]
@iamrahul823 I can assign this too, but will recommend finishing this first. Once you submit a PR, tag me again on this issue again

**Comment by thekhaleed** [comment]
Hello @yashikakhurana. I am an outreachy applicant, please assign me this.

**Comment by yashikakhurana** [comment]
@thekhaleed assigned, thank you

**Comment by yashikakhurana** [comment]
@thekhaleed if you are not working, I have to remove your assignment, let me know otherwise tomorrow I will remove your assignment as other applicants are waiting to get assigned, kindly submit a PR.

**Comment by thekhaleed** [comment]
Hello @yashikakhurana, I am working on it

**Comment by yashikakhurana** [comment]
@thekhaleed checking again, are you still working on this?

**Comment by thekhaleed** [comment]
I texted you about it in Element letting you know the opt-out expressions are not there and the desktops being talked about in the code are latest versions of desktops. There is no desktop 83 anymore.

**Comment by yashikakhurana** [comment]
@thekhaleed we need to remove opt out `app.shield.optoutstudies.enabled`, you see in the issue description it is pointing out to particular commit, now we can get rid of those added lines, does it make sense now?

**Comment by Prithvi-Chanchal** [comment]
@yashikakhurana I think the added lines are already removed in the current version, I checked the files mentioned in the commit and could not find any of these lines. could you check that?

**Comment by yashikakhurana** [comment]
@Prithvi-Chanchal true, I double checked this is no longer valid, thank you for noticing that.
